packages/react-storage/src/components/StorageManager/hooks/useUploadFiles/resolveFile.ts

```diff
@@ -0,0 +1,31 @@
+import { isFunction } from '@aws-amplify/ui';
+import { StorageManagerProps } from '../../types';
+
+interface ProcessFileParams extends Pick<StorageManagerProps, 'processFile'> {
```

Comment:
Can we use the `ProcessFileParams` defined [here](https://github.com/aws-amplify/amplify-ui/pull/3798/files#diff-41fc325b42b2c3ea38b8bb65e8f03aba29456c1a904f8a8e7a87b9329730047aR35)?